script.js

```diff
@@ -0,0 +1,114 @@
+          FINALLY RELEASED ON TV:
+          Two brothers. In a van. And then a meteor hit. And they ran as fast as they could. From giant cat-monsters. And then a giant tornado came. And that's when things got knocked into twelfth gear...
+            A Mexican...armada shows up. With weapons made from to- tomatoes. And you better betch'ur bottom dollar that these two brothers know how to handle business.
+            In! 'Alien...Invasion Tomato Monster Mexican Armada Brothers...Who Are Just Regular Brothers Running...in a Van from an...Asteroid and All Sorts of Things: The Movie'.
+            Hold on! There's more. Old women are comin'! And they're also in the movie and they're gonna come...and cross...attack...these two brothers. But let's get back to the brothers because they're- they have a strong bond! You don't wanna know about it here, but I'll tell you one thing.
+            The Moon. It comes crashing into Earth! And whaddya do then? It's two brothers and I- and...and they're gonna...it's called 'Two Brothers' ...'Two Brothers'...it's just called 'Two Brothers'!
+        -How They Do It
+          This week's episode: The Plumbus.
+          Make sure you add enough fleeb.
+        -Gazorpazorpfield
+          Man, Gazorpazorpfield sure was pissed off in this episode.
+
+Jerry:
+-Wow, really? You picked the wrong conceptual horse, here. Ok, how many jobs can you help Jerry apply to?
+  for each number listed, each job application will be denied:
+  for (var i = 1, i <= jobsApplied; i++) {
+  alert("Job application: "+i+" STATUS: DENIED")
+}
+
+
+
+*/
+
+var userName = prompt("Welcome to the Rick & Morty CYOA! Enter your name to begin:");
+var hang = prompt("Hello, "+userName+". Who would you like to hang with today? (R) Rick, (M) Morty, (J) Jerry").toUpperCase()
+//var hang = hang1.toUpperCase()
+
+//Code for Rick outcome below
+if (hang == "R") {
+    var rickChoice = prompt("It's a slow summer day, Rick wants to know if you want to help him (F) fix the spaceship battery or (B) spend the day gambling shmeckles at Blips and Chitz.").toUpperCase()
+//Fix battery
+    if (rickChoice == "F") {alert("You and rick are transported into the car battery microverse. A mad scientist in the battery microverse has invented a microverse to power his world. You enter the microverse's microverse... After countless Inception-style levels, you resolve the battery issue. Rick awards you 200 shmeckles.")
+    }
+//Blips and Chitz
+    else if (rickChoice == "B") {
+        var roy1 = prompt("You enter the Roy simulation game. To begin the game, what year do you want Roy to be born?")
+        var roy = parseInt(roy1)
+          if (roy===1989) {alert("You score high enough for first place! Everyone at Blips and Chitz loves you and you've earned Rick's respect. +1,000,000 Shmeckles.")}
+          else if (roy <=1900) {alert("Life was tough back then. You score 89,687, good enough for 162nd place. Rick laughs at your poor performace.")}
```

Comment:
If your `if`, `else if`, or `else` is all on one line, you can actually write it like this:
```
if(x === 3) doSomething();
else if(x === 4) doAnotherThing();
else doSomethingElse();
```

You can also mix and match:
```
if(x === 3){
  doSomething();
}else doSomethingElse();
```